docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -74,3 +74,41 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
+
+## Alerting: Grafana OnCall integration
+
+<!-- George Robinson -->
+
+_Generally available in Grafana_
```

Comment:
Which editions of this Grafana is this available in? OSS, Enterprise, all of Cloud, some sub-set of Cloud? Same question for the other entries.

Reply:
All - it was released and added to cloud release notes a while ago - and is now generally available with 10.2